boardgame-ui/src/components/games/AddEditGameForm.vue

```diff
@@ -282,6 +289,7 @@ export default {
       this.game = await Game.fetch(this.idGame);
 
       this.searchString = this.game.board_game.name;
+      this.eventSearchString = this.events.find(event => event.id == this.game.id_event).name;
```

Comment:
```suggestion
if(this.game.id_event) {
      this.eventSearchString = this.events.find(event => event.id == this.game.id_event).name; // should actually be something like this.selectedEvent = this.events.find(event => event.id == this.game.id_event)
}
```
Otherwise, fails when trying to edit game outside event.